src/drop-down-menu.jsx

```diff
@@ -144,19 +153,11 @@ const DropDownMenu = React.createClass({
         whiteSpace: 'nowrap',
       },
       rootWhenOpen: {
-        opacity: 1,
+        opacity:1,
```

Comment:
I want to enforce a rule with eslint so that we always have one space after `:`.
What do you think? If you agree could you remove this change, thanks.